lib/libpng/CMakeLists.txt

```diff
@@ -5,9 +5,7 @@
 
 # This only has effect in this directory (lib/png).
 include_directories(include/)
-if(WIN32)
-	include_directories(../zlib/include/)
-endif()
```

Comment:
This test should probably just be changed to `if(UseInternalZlib)`. It will take care of the `WIN32` case where it is set to ON by default and the Linux case when internal libraries are requested.